client/src/usePaginatorControls.js

```diff
@@ -0,0 +1,68 @@
+import { useState } from "react";
+import MagicLabeler from "./MagicLabeler";
+
+const usePaginatorControls = (currentPage, getMagicLabel, setLabel) => {
+    const [labelInput, setLabelInput] = useState("");
+    const getControlsLabel = (useMagic) => {
+        if (typeof useMagic === "undefined") {
```

Comment:
Could we do null coalescing below (i.e. replace `useMagic` with `(useMagic ?? true)`) to simplify this code?

Reply:
From what I remember, this behaves differently whether it is undefined or false.

Reply:
I think there is different behavior depending on whether the label is false or undefined, but this particular code is just designed to default the useMagic flag parameter to true if it is not provided (probably because Javascript didn't support default parameter values when I first wrote this code many years ago).